floor/lib/src/adapter/insertion_adapter.dart

```diff
@@ -42,7 +42,7 @@ class InsertionAdapter<T> {
       );
     }
     await batch.commit(noResult: true);
-    _changeListener?.add(_entityName);
+    _changeListener?.add({_entityName});
```

Comment:
Actually, the insertion_adapter also needs lambdas because of the onconflict:update issue... I thought I committed that already but it seems like I didn't...

Reply:
ah right, I didn't do this because the `onConflict` clause can be different for each call of `insertX`, altering the entities which have to be notified... And I could not decide what is best.

There are basically three main options here:
1. always assume `onConflict:Update` if it occurs even once for a `InsertionAdapter.insert` call
2. give the `InsertionAdapter` two different sets/lambdas via the constructor and it will choose which one to use for triggering updates, based on the `onConflict` clause given in the `InsertionAdapter.insert` call
3. insert the lambda/set via the invocation of the `InsertionAdapter.insert` call and not in the constructor

I'm not sure which is best here, but 1 seems to be suboptimal to me.